src/views/Governance/Governance.tsx

```diff
@@ -92,16 +93,18 @@ const Governance: React.FC = () => {
               </StyledProposalContentInner>
             </Box>
             <Spacer size="sm" />
-            {proposals && (
+            {proposals ? (
               <Surface>
                 {proposals.map((prop, i) => {
                   if (i === 0) {
                     return <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />;
                   } else {
-                    return [<Separator />, <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />];
+                    return [<Separator key={"seperator" + i}/>, <ProposalEntry key={prop.hash} prop={prop} onVote={onVote} onRegister={onRegister} />];
```

Comment:
any reason adding this here?

Reply:
please check this document
https://reactjs.org/docs/reconciliation.html#keys